packages/patterns-react/src/patterns/CardArray/CardArrayItem.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import { Button } from 'carbon-components-react';
+import { ArrowRight20, ArrowDown20, Pdf20 } from '@carbon/icons-react';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Card Array Item Object
+ *
+ * @param {object} props props object
+ * @param {string} props.title Card Title
+ * @param {string} props.copy Card copy
+ * @param {object} props.link Card link object
+ * @returns {*} JSX Card Array Item Object
+ */
+const CardArrayItem = ({ title, copy, link }) => {
+  return (
+    <div
+      data-autoid={`${stablePrefix}--cardarray-item`}
+      className={`${prefix}--cardarray-item`}>
+      <h4 className={`${prefix}--cardarray-item__title`}>{title}</h4>
+      <p className={`${prefix}--cardarray-item__content`}>{copy}</p>
+      <div className={`${prefix}--cardarray-item__link`}>
+        <Button
+          className={`${prefix}--cardarray-item__link__button`}
+          href={link.href}
+          target={link.target}
+          kind="primary">
+          {_renderIcon(link.icon)}
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+/**
+ * Renders the right icon based on the icon variable
+ *
+ * @param {string} icon Icon name string
+ * @returns {*} Icon JSX object
+ */
+const _renderIcon = icon => {
+  switch (icon) {
+    case 'ArrowRight':
+      return <ArrowRight20 fill="white" />;
+    case 'ArrowDown':
+      return <ArrowDown20 fill="white" />;
+    case 'Pdf':
+      return <Pdf20 fill="white" />;
+    default:
+      return <ArrowRight20 fill="white" />;
+  }
+};
```

Comment:
Are these the only icons we are allowing for this, or do we need to open this up to any type of icon? If it's any icon, we would have to think about how those are imported. If it's just these three, we'd need to have these documented somewhere on what are the options available to use.

Reply:
Reached out to Jill and Lila about that^

As from Lila: "I think for this pattern all cards would only use the arrow right icon"

So we can simplify this to only use the `ArrowRight` icon

Reply:
Thanks for checking this @annawen1 , I'll use the right arrow in this case.